web/pdf_viewer.js

```diff
@@ -740,12 +740,15 @@ class PDFViewer {
       // getAllText and we could just get text from the Selection object.
 
       // Select all the document.
-      const savedCursor = this.container.style.cursor;
-      this.container.style.cursor = "wait";
-
-      const interruptCopy = ev =>
-        (this.#interruptCopyCondition = ev.key === "Escape");
-      window.addEventListener("keydown", interruptCopy);
+      const { classList } = this.viewer;
+      classList.add("copyAll");
```

Comment:
Why not using this one https://github.com/mozilla/pdf.js/blob/e3caa3c6eeda90099dd03b451b6b4788746455b3/web/viewer.css#L227 instead ?

Reply:
I only tried to re-factor existing code in this patch, and there's also the question if you really want to "disable" the entire viewer while viewer copying runs since that's the effect of the referenced CSS rule (via the `z-index`)?

Furthermore, we really cannot access `document.body` from the `PDFViewer` class.